<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoefficientManager</class>
 <widget class="QDialog" name="CoefficientManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>590</width>
    <height>629</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>629</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Noskaņot koeficientus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCoefficientsButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Saglabāt novērt. funkijas koeficientus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Atgriezties bez izmaiņu saglabāšanas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="locale">
    <locale language="Latvian" country="Latvia"/>
   </property>
   <property name="text">
    <string>Faila nosaukums, no kura nolasīt koeficientus (mapē Optimization_Results) :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileNameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Latvian" country="Latvia"/>
   </property>
  </widget>
  <widget class="QPushButton" name="readCoeffiecientsFromFileButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>88</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Nolasīt koeficientus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Parametri:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Koeficienti:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>1) Attālums starp spēlētāja pirmo zirgu un startu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>2) Attālums starp spēlētāja otro zirgu un startu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>3) Vai vēl nebija izdarīts gājiens ar spēlētāja pirmo zirgu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>4) Vai vēl nebija izdarīts gājiens ar spēlētāja otro zirgu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>5) Attālums starp spēlētāja pirmo zirgu un līderi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>6) Attālums starp spēlētāja otro zirgu un līderi:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>7) Vai spēle pēc gājiena beidzas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>8) Vai spēlētāja pirmais zirgs finišē:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>9) Vai spēlētāja otrais zirgs finišē:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>10) Attālums starp spēlētāja 1 zirgu un spēlētāja 2 zirgu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>11) Attālums starp spēlētāja 1 zirgu un cita spēlētāja 1 zirgu,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>     kas apsteidz visu citu spēlētāju pirmos zirgus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>12) Attālums starp spēlētāja 1 zirgu un cita spēlētāja 2 zirgu,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>     kas apsteidz visu citu spēlētāju otros zirgus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>13) Attālums starp spēlētāja 2 zirgu un cita spēlētāja 1 zirgu,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>     kas apsteidz visu citu spēlētāju pirmos zirgus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>     kas apsteidz visu citu spēlētāju otros zirgus:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>14) Attālums starp spēlētāja 2 zirgu un cita spēlētāja 2 zirgu,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>15) Attālums starp spēlētāja 1 zirgu un tuvāko nākamo zirgu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>16) Attālums starp spēlētāja 2 zirgu un tuvāko nākamo zirgu:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_1_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_2_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_3_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_4_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_5_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_6_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_7_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_8_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_9_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_10_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>340</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_11_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_12_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_13_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_14_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_15_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>520</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coef_16_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>540</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
